src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -1373,7 +1373,7 @@ public void actionPerformed(ActionEvent e) {
 	private boolean isValidPaymentDate(GregorianCalendar datePay) {
 		GregorianCalendar now = new GregorianCalendar();
 		GregorianCalendar lastPay = new GregorianCalendar();
-	    if (payItems.size() > 0) { 
+	    if (!payItems.isEmpty()) {
 	    	lastPay = payItems.get(payItems.size()-1).getDate();
 		} else {
 			lastPay = billDate;
```

Comment:
Strange indentation

Reply:
I'll fix it; it is most likely a tabs vs. spaces issue.

Reply:
Reformatted the entire method so it should be consistent.

Reply:
Thanks!